### DISCUSSION TITLE
Como pagar menos o seu programador.

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#389] [Discussão]
Oi pessoal, sempre que vejo um artigo interessante no pocket suggestions e vejo que ninguem postou aqui, eu tento trazer pro grupo, vi esse recentemente, que fala ironicamente sobre uma realidade de mercado:

http://www.yegor256.com/2016/12/06/how-to-pay-programmers-less.html

Queria saber a opinião de vocês.

E queria saber também se existe algum tópico aqui que direciona programadores a serem bons freelancers e nomades.

Eu tenho me esforçado para trabalhar em empresa e até gosto, mas meu ideal de vida mesmo é trabalhar remoto de onde quiser.

Se alguem tiver qualquer material falando sobre que não seja sites como o Nomades Digitais, e coisas motivacionais, do tipo, depois que eu consegui tudo é facil. 

Vi a galera falando do upwork, mas vi que lá 99% são trabalhos muito mal remunerados.
Sei da possibilidade de temas wordpress também, ou de ter clientes fixos, mas wordpress é um mercado bem complicado, dependendo dos clientes que você tem.

**Comment by barbier** [comment]
Enquanto eu lia, o alerta de sarcasmo começou a apitar loucamente, mas pode ter sido comigo...
Inclusive eu trabalhei numa empresa em que parece que seguia pontos dessa cartilha antes de existir...

**Comment by rubenmarcus** [comment]
sim é sarcasmo o artigo

**Comment by adeonir** [comment]
tem sarcasmo em todos artigos do cara...

**Comment by felipefialho** [comment]
Não sei se é uma pratica tão comum no mercado.

Lógico que está cheio de startups e empresas espertalhonas, mas com a expansão ta comunicação entre desenvolvedores, empresas que praticam essas abordagens ficam marcadas e tem muito mais dificuldade para contratar bons profissionais.

Mas claro, é sempre bom ficar atento. 

Por hora, o texto é awesome 😹 😹 😹

**Comment by kazzkiq** [comment]
As empresas sempre vão focar em **pagar menos** (cortar custos), e os empregados sempre vão focar em **ganhar mais**. Cada um puxa a sardinha pro seu lado, funciona assim desde que a Terra Média foi ocupada pelos primeiros elfos.

O que sempre define os valores, caso não exista uma interferência externa, é a **lei da oferta e da demanda**. Por sorte, a nossa área ta com uma demanda crescente nos últimos anos (e provavelmente só vai aumentar daqui pra frente), então no caso de nós, devs front-end, fica mais fácil filtrar empresas boas que paguem bem, de empresas ruins. Por conta da alta demanda, a gente acaba tendo esse poder de escolha.

A treta toda do "baixo salário" em sites de freelas como o UpWork é justamente a altíssima oferta de mão-de-obra. Você tem milhares de devs do mundo todo (principalmente indianos, quem já entrou lá sabe que os caras comandam esses sites), que concorrem com você, topando receber menos pelos mesmos jobs. Aí mesmo numa área em alta como a nossa os salários acabam caindo.

O segredo ao meu ver, é você montar um portfólio bacana, e ir criando uma carteira de clientes (no Brasil mesmo) que criem uma espécie de "vínculo de confiança" com o seu trabalho. A partir desse momento, quando essas empresas confiarem no seu trampo, elas sempre vão te procurar antes de qualquer outra pessoa, por conta do passado positivo de jobs bem feitos. Dessa forma você começa a criar uma certa demanda pelo seu trabalho e consequentemente vai poder escolher os que paguem mais, aumentando seu salário como freelancer.

Se montar essa carteira de clientes é fácil? **Obvio que não**. Eu acho que temos freelas aqui no fórum com muito mais experiência nessa área que podem dar uma ideia de como você vai ter que ralar no inicio pra conseguir juntar uma clientela "fiel". Mas eu, como freelancer esporádico, acredito que depois de criado um nome no mercado, o valor dos seus trampos inevitavelmente vai subir. 

*Sow your reputation, to reap the big money later*

**Comment by mksanti** [comment]
@kazzkiq falou tudo !

**Comment by leandroruel** [comment]
@kazzkiq :clap:
